pkg/controller/infinispan/autoscale.go

```diff
@@ -0,0 +1,204 @@
+package infinispan
```

Comment:
@rigazilla wdyt to place it in separate package outside the controller, for example **autoscale**?

Reply:
this creates some `import *` issues and I don't know if the autoscale has some utility outside from infinispan atm

Reply:
Oh, yes, cyclic dependency issue.